source/common/buffer/buffer_impl.cc

```diff
@@ -25,7 +26,7 @@ void OwnedImpl::addImpl(const void* data, uint64_t size) {
   bool new_slice_needed = slices_.empty();
   while (size != 0) {
     if (new_slice_needed) {
-      slices_.emplace_back(size);
+      slices_.emplace_back(Slice(size, account_));
```

Comment:
I think this creates a temporary, then moves it into `slices_`. Does it compile as `slices_.emplace_back(size, account_);`? I can't remember if that's the syntax to get it to construct-in-place, or if something else is needed.

Reply:
Oh right, I forgot that this wasn't a "real" stl container. Yeah, this is fine, same as it was previously.

Reply:
It may be worth implementing the 2 argument emplace_back or perfect forwarding, but Slice and move is fine for now.